apps/api/src/app/inbox/inbox.controller.ts

```diff
@@ -225,7 +225,7 @@ export class InboxController {
   async updateGlobalPreference(
     @SubscriberSession() subscriberSession: SubscriberEntity,
     @Body() body: UpdatePreferencesRequestDto
-  ): Promise<ISubscriberPreferences> {
+  ): Promise<InboxPreference> {
```

Comment:
created a new type for the preferences as the old one doesn't satisfy our needs and is used in other old NC use-cases